meson_options.txt

```diff
@@ -65,6 +65,16 @@ option('install-docs',
        value : true,
        description : 'Install documentation for API and specification'
 )
+option('tests',
+       description: 'Build the test suite',
+       type: 'boolean',
+       value: true
+)
```

Comment:
Tests are always optional, you never *have* to run them. What's the goal here? Just to avoid compiling the test programs by default?

I would suggest instead setting `build_by_default: false` for test progs. Meson 1.7 won't build them by default anymore, if you do so.

Reply:
Yes 

1.7 is too new for me to benefit from it.